appveyor.yml

```diff
@@ -0,0 +1,29 @@
+---
+version: "{build}"
+skip_tags: true
+branches:
+  only:
```

Comment:
Okay, gotcha. Just a heads-up, this is a work-in-progress PR since I'm pushing stuff to the PR and experimenting/fixing errors as Appveyor reports them in the builds. They do seem to support local testing in the same way you can work with Travis builds in a local container but I'd need to boot up a Windows VM so this seems easier.